kinetic/distribution.yaml

```diff
@@ -7973,6 +7973,26 @@ repositories:
       url: https://github.com/ros/nodelet_core.git
       version: indigo-devel
     status: maintained
+  noid-ros-pkg:
+    doc:
+      type: git
+      url: https://github.com/seed-solutions/noid_ros_pkg.git
+      version: master
+    release:
+      packages:
+      - noid_typef_bringup
+      - noid_typef_description
+      - noid_typef_moveit_config
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://github.com/seed-solutions/noid_ros_pkg-release.git
+      version: 0.0.5-1
+    source:
+      test_pull_requests: true
```

Comment:
Just to double check; did you follow the instructions on http://wiki.ros.org/buildfarm/Pull%20request%20testing to allow Jenkins access to your repository?  Pull request testing won't work without that.

Reply:
@clalancette  
I added "ros-pull-request-builder" collaborator of the package.

Reply:
@clalancette 
Sorry, I forgot to change the license from EPL to BSD.
So I'm going to change the license when next update.
Do you have a problem with this?

Reply:
Sorry for the delay.  No, this is fine, thanks for following up.